libraries/grubconf.rb

```diff
@@ -0,0 +1,7 @@
+class GrubConf < Inspec.resource(1)
+  name 'grub_conf'
+
+  def locations
```

Comment:
🎉 Thank you for moving the config to the resource!

Reply:
Done :)